build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/ProjectExtensions.kt

```diff
@@ -19,7 +19,22 @@ package com.google.samples.apps.nowinandroid
 import org.gradle.api.Project
 import org.gradle.api.artifacts.VersionCatalog
 import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.api.file.Directory
+import org.gradle.api.provider.Provider
 import org.gradle.kotlin.dsl.getByType
 
 val Project.libs
     get(): VersionCatalog = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
+/**
+ * Create new Directory by [dir] name in the root build directory.
```

Comment:
The naming is misleading as this does not create anything. Can you change this?

Reply:
Also, please mention that it uses the 'build' directory.
Imo, this should probably not be extracted as an extension here, but could be simply declared as an inlined extension where it is actually used.

Reply:
I'll change it into inlined extension.